src/components/tabsMessagesFriends/Friends.tsx

```diff
@@ -1,76 +1,95 @@
-import * as React from 'react'
+import React from 'react'
 import { Box, Typography } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
-import friends from './friendsProfile.json'
 import { UserProfileData } from '../../types/UserProfileData'
+import NoNewMatches from './NoNewMatchesOrMessages'
+import { useNewFriendsList } from 'hooks/useFriendsList'
+import theme from 'styles/createTheme'
 
 interface FriendsProps {
   onClick: (userProfileData: UserProfileData) => void
+  selectedFriend: UserProfileData
 }
 
-const Friends: React.FC<FriendsProps> = ({ onClick }) => {
+const Friends: React.FC<FriendsProps> = ({ onClick, selectedFriend }) => {
   const { classes } = useStyles()
+
+  const { data: userFriends } = useNewFriendsList()
+
   const handleClick = (id: string) => {
-    const friendsData = friends.find((element) => element.id == id)
+    const friendsData = userFriends?.find(
+      (element: UserProfileData) => element.id == id
+    )
     onClick(friendsData as UserProfileData)
   }
+
   return (
-    <Box className={classes.friendsBlock}>
-      {friends.map((element) => (
-        <Box
-          id={element.id}
-          key={element.id}
-          className={classes.friendsPhotos}
-          onClick={() => handleClick(element.id)}
-        >
-          <img
-            src={element.photo[0].src}
-            alt="photo"
-            className={classes.smallPhoto}
-          />
-          <Typography className={classes.textOnPhoto}>
-            {element.firstName} {element.lastName}, {element.age}
-          </Typography>
+    <>
+      {userFriends?.length == 0 ? (
+        <NoNewMatches text="You don’t have new matches." />
+      ) : (
+        <Box className={classes.friendsBlock}>
+          {userFriends?.map((element: UserProfileData) => (
+            <Box
+              id={element.id}
+              key={element.id}
+              className={`${classes.friendsPhotos}  ${
```

Comment:
FYI, it's opinionated thingy of cause
`      className={classnames({
        [classes.friendsPhotos]: true,
        [classes.fotoBorder]: element.id === selectedFriend.id,
        ...
      })}`
      you can add package classnames

Reply:
Done